roles/pulp_common/tasks/install_pip.yml

```diff
@@ -209,14 +209,13 @@
 
     - name: Install pulpcore via PyPI
       pip:
-        name: "pulpcore{{ '['+ pulp_install_object_storage +']' if pulp_install_object_storage != 'filesystem' else '' }}"
+        name: "pulpcore{{ '['+ pulp_install_object_storage +']' if pulp_install_object_storage != 'filesystem' else '' }}{% if pulpcore_version | wordcount >2 %}=={{ pulpcore_version }}{% else %}~={{ pulpcore_version }}.0{% endif %}"
         # pulp_upgrade true/false -> ansible pip module latest/present ->
         # pip command "install --upgrade"/"install"
         # Commented out because:
         # "version is incompatible with state=latest"
+        state: '{{ (pulpcore_update == true) | ternary("latest", "present") }}'
         # But we still need to handle upgrading dependencies.
```

Comment:
I'm willing to bet that the dependencies will be upgraded if state=latest.

Reply:
This comment needs to be rephrased and moved above the line it talks about (state), assuming dependencies will be upgraded.